modules/apps/web-experience/export-import/export-import-service/src/main/java/com/liferay/exportimport/controller/PortletExportController.java

```diff
@@ -333,14 +333,14 @@ public void exportPortlet(
 			exportPortletPreferences(
 				portletDataContext, portletDataContext.getCompanyId(),
 				PortletKeys.PREFS_OWNER_TYPE_COMPANY, false, layout, plid,
-				portlet.getRootPortletId(), portletElement);
+				portlet.getPortletId(), portletElement);
```

Comment:
I see, since in 7 the embedded portlet's configuration has a group scope, it's not being exported. However, by making this change you will cause the regular page portlets' configuration to not be exported. Probably the solution is to add a new call to exportPortletPreferences, right?